### DISCUSSION TITLE
Multiple accounts

### DISCUSSION THREAD

**Discussion by pawmasz** [community/community#6599] [Enterprise] [answered]
Hello, is it possible to add feature to log in more than one account in GitHub mobile app? It would be nice to log in with private, and enterprise account in the same place.

**Comment by pawmasz** [comment, marked as answer]
Turns out this is a feature already. But it would be nice to have simpler switch between the accounts in main view, like it is in OneDrive for example, and not need to go into settings to switch.

**Reply by aconsuegra** [reply]
Thanks for the suggestion! I'll make sure to pass the feedback to our design team.

**Reply by brianlovin** [reply]
@pawmasz on both iOS and Android, we have a shortcut to open the account switch by long-pressing on the Profile tab in the tab bar.

**Reply by pawmasz** [reply]
Wow! Didn't know about that. Thank you! Great support guys 😊
